packages/cli/test/integration/generators/app.integration.js

```diff
@@ -165,6 +167,86 @@ describe('app-generator with --applicationName', () => {
   });
 });
 
+/** For testing the support of tilde path as the input of project path */
+describe('app-generator with tilde project path', () => {
+  const rootDir = path.join(__dirname, '../../../..');
+  const projPath = path.join(rootDir, 'sandbox/tilde-path-app');
+
+  /** NOTICE: if this test runs outside of the current user's $HOME directory,
+  then pathWithTilde will be identical to projPath. Means the tests will pass
+  without testing the ~ support. This could come up on CI systems where
+  tests are run inside /tmp, for example. */
+  if (process.env.CI && !process.env.DEBUG && tildify(projPath) === projPath) {
+    // warn the user that it doesn't test the support of `~`
+    console.error(
+      "WARNING: couldn't test the support of tilde in path under this test environment even though the test might pass",
```

Comment:
This fails CI - see https://ci.appveyor.com/project/strongloop/loopback-next/builds/25882737. We don't expect to have console output.

Reply:
Is there any way to warn the user if they run `npm t` under such test environment?

Reply:
I'm not sure why we want to warn the user.

Reply:
Or I can just leave a comment if that's better @raymondfeng